Detectors/FIT/common/simulation/src/Digitizer.cxx

```diff
@@ -18,26 +18,88 @@
 #include <algorithm>
 #include <cassert>
 #include <iostream>
+#include <optional>
 
 using namespace o2::fit;
 //using o2::fit::Geometry;
 
 ClassImp(Digitizer);
 
-double get_time(const std::vector<double>& times, double signal_width)
+double sinc(const double x)
 {
-  TH1F hist("time_histogram", "", 1000, -0.5 * signal_width, 0.5 * signal_width);
-  auto signalForm = [](double x) {
-    return -(exp(-0.83344945 * x) - exp(-0.45458 * x));
-  };
-  for (auto time : times)
-    for (int bin = hist.FindBin(time); bin < hist.GetSize(); ++bin)
-      if (hist.GetBinCenter(bin) > time)
-        hist.AddBinContent(bin, signalForm(hist.GetBinCenter(bin) - time));
-  double maximum{ hist.GetMaximum() };
-  int binfound{ hist.FindFirstBinAbove(maximum * 0.4) };
-  //  std::cout<<"@@@@@!!! max "<<maximum<<" binfound "<<binfound<<" return "<<hist.GetBinCenter(binfound)<<std::endl;
-  return hist.GetBinCenter(binfound);
+  if (x == 0)
+    return 1;
+  return sin(x) / x;
+}
+double signalForm_i(double x)
+{
+  return -(exp(-0.83344945 * x) - exp(-0.45458 * x)) * (x >= 0) / 7.8446501; // Maximum should be 7.0/250 mV
```

Comment:
if this function is defined for x>=0 and negative x can be provided, it is more efficient to do
``return x<0 ? 0 : -(exp(-0.83344945 * x) - exp(-0.45458 * x))/7.8446501;``
Also this can be inlined.